### DISCUSSION TITLE
bug found on website

### DISCUSSION THREAD

**Discussion by singhabhinendra** [sanjay-kv/Open-source-Practice#1145] [Q&A]
In Headline

**Comment by Aryan-9488** [comment]
elaborate the bug plz

**Comment by alexcerezo** [comment]
I thing that you should explain it more or mark this problemas answered.

**Comment by Akash-Gupta-git** [comment]
where is website ?

**Comment by codeftsaikat** [comment]
You should explain it more for help you.

**Comment by Randil-Hasanga** [comment]
Can you give more information?

**Comment by viswanathr73** [comment]
Explain more about the bug !

**Comment by Himavanthkumar** [comment]
Please provide the link to your website and mark this as Answered. Thankyou!
